ownCloud/Tools/VendorServices.swift

```diff
@@ -146,7 +146,7 @@ extension VendorServices : OCClassSettingsSupport {
 
 	static func defaultSettings(forIdentifier identifier: OCClassSettingsIdentifier) -> [OCClassSettingsKey : Any]? {
 		if identifier == .app {
-			return [ .isBetaBuild : false, .showBetaWarning : false, .enableUIAnimations: true ]
+			return [ .isBetaBuild : true, .showBetaWarning : true, .enableUIAnimations: true ]
```

Comment:
@hosy Shouldn't this be reverted for official release?

Reply:
Yes, at the moment milestone/1.2 does point to the official master. Will be fixed!